client/src/Redux/Thunk/getRoomNoAsync.js

```diff
@@ -5,7 +5,8 @@ import {setERROR} from "../Action/errorAction";
 export function getRoomNumberAsync(userName, timeControl, timeIncrement) {
     return async (dispatch, getState) => {
             try {
-                const response = await axios.get((process.env.REACT_APP_BACKEND_URL || 'http://localhost:5001') + '/createGame',
+                const response = await axios.get(('http://localhost:5001') + '/createGame',
+                // const response = await axios.get((process.env.REACT_APP_BACKEND_URL || 'http://localhost:5001') + '/createGame',
```

Comment:
I tried the line of code in red which I commented out, but it will stick at the first deployed URL and won't go to the second URL - the localhost. I need to test the app with the localhost so I commented out the line of code in red. Is there something I do not configure correctly?

Reply:
It worked for me on new-branch. You can take them out for testing. But please make sure put them back otherwise deployment will have an issue.